nikola/nikola.py

```diff
@@ -1984,6 +1984,20 @@ def create_hierarchy(cat_hierarchy, parent=None):
         # Next, flatten the hierarchy
         self.category_hierarchy = utils.flatten_tree_structure(root_list)
 
+    def sort_posts(self, posts, lang=None):
```

Comment:
It is intentionally not a static method, because it might want to use the site's config in the future.